lib/blueprinter/serializers/local_method_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class Blueprinter::LocalMethodSerializer < Blueprinter::Serializer
+  def serialize(field_name, _object, options = {})
+    options[:local_methods][field_name]
```

Comment:
If we change to this:
`options[:local_methods][field_name].call(_object, options)`

We should get the ability to interact with both the object AND any options passed into the blueprint, right?

Reply:
that's a great idea

Reply:
@dlcarter What do you see as useful for exposing the `options` variable through the block right now?

Right now, `render` does not accept an option args. 

We could allow `local_method` to accept `options` args, but do you see that as being useful right now? Or should that wait for #33 when we work on exposing options arguments for `render`?